src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -689,9 +702,9 @@ private static Collection<Token> validateReplacementBootstrapTokens(TokenMetadat
 
     public synchronized void checkForEndpointCollision(UUID localHostId, Set<InetAddressAndPort> peers) throws ConfigurationException
     {
-        if (Boolean.getBoolean("cassandra.allow_unsafe_join"))
+        if (ALLOW_UNSAFE_JOIN.getBoolean())
         {
-            logger.warn("Skipping endpoint collision check as cassandra.allow_unsafe_join=true");
+            logger.warn("Skipping endpoint collision check as " + ALLOW_UNSAFE_JOIN.getBoolean() + "=true");
```

Comment:
you meant `ALLOW_UNSAFE_JOIN.key()`, no?

Reply:
Good catch. My bad.
Fixed.